_pytest/fixtures.py

```diff
@@ -615,6 +615,18 @@ def scopemismatch(currentscope, newscope):
     return scopes.index(newscope) > scopes.index(currentscope)
 
 
+def strip_invocation_scope_suffix(name):
+    """Remove the invocation-scope suffix from the given name.
+
+    Invocation scope fixtures have their scope in the name of the fixture.
+    For example, "monkeypatch:session". This function strips the suffix
+    returning the user-frienldy name of the fixture.
+    """
+    if ':' in name:
```

Comment:
``str.split()`` returns the full str if it doesn't contain the separator.

So this function could be

```python
def strip_invocation_scope_suffix(name):
    return name.split(':')[0]
```